### ISSUE TITLE
Test 6.9: assumes reply has content

### ISSUE THREAD

**Issue by daurnimator** [summerwind/h2spec#43]
```
  6.9. WINDOW_UPDATE
    × Sends a WINDOW_UPDATE frame
      - The endpoint is expected to send the DATA frame based on the window size.
        Expected: DATA frame
          Actual: Error: The length of DATA frame is invalid.
    ✓ Sends a WINDOW_UPDATE frame with an flow control window increment of 0
    ✓ Sends a WINDOW_UPDATE frame with an flow control window increment of 0 on a stream
    ✓ Sends a WINDOW_UPDATE frame with a length other than a multiple of 4 octets
```

The endpoint I tested had a 0 length document.

**Comment by summerwind** [comment]
Thanks for the report. Please use the document of 10 or more bytes of size. 
I'm going to change the code to skip this test if the document size is less than 10 bytes.

**Comment by summerwind** [comment]
Fixed in c4e383421fb7fd265f5c5e3392341e301f86ff67.

**Comment by daurnimator** [comment]
Thanks.
